### PR TITLE
Version update

### PR DESCRIPTION
**Pull request by PavanBilagiHalodoc**
1. Kotlin version updated to 1.4.30

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        kotlin_version = '1.3.50'
+        kotlin_version = '1.4.30-RC'
```
Comment (tibagni) [inline build.gradle:3]: Seems like latest version is 1.4.30 already (https://github.com/JetBrains/kotlin/releases/tag/v1.4.30) - without RC
I am ok to update kotlin version as long as all tests pass and there is no issue in normal execution, but I think we should use 1.4.30 (without RC)

**Review by tibagni — CHANGES_REQUESTED** [review]
Please use 1.4.30 (without RC)

File: build.gradle (hunk 1 above)
Comment (PavanBilagiHalodoc) [reply build.gradle]: Changed to  1.4.30 @tibagni

**Comment by PavanBilagiHalodoc** [comment]
@tibagni Requested changes are pushed.

**Review by tibagni — APPROVED** [review]

**Pull request merged**